doc/ug/dv_methodology/index.md

```diff
@@ -465,7 +465,32 @@ It is recommended to follow these guidelines when collecting coverage:
 *  Collect all coverage metrics (except toggle based on above bullet) on the DUT and all of its non-pre-verified sub-modules
 *  Treat pre-verified sub-modules as ['black-box'](#integration-testing) in terms of coverage collection
 
-### X-Propagation (Xprop)
+### Unreachable Coverage Analysis
+
+Instead of manually reviewing coverage reports to find unreachable code, we use VCS UNR to generate a UNR exclusion file which lists all the unreachable codes.
+VCS UNR (Unreachability) is a formal solution that determines the unreachable coverage objects automatically from simulation.
+The coverage config file ([cover.cfg](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/vcs/cover.cfg)) used with `-cm_hier` in simulation will be used as UNR input to cater our [Coverage Collection Guidelines](#Coverage Collection Guidelines), and common exclusion file ([common_cov_excl.el](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/vcs/common_cov_excl.el)) can be also supplied to UNR to avoid generating reviewed common unreachable coverage.
```

Comment:
We already talk about 'coverage hierarchy file' in the previous section. You can just say `The same coverage hierarchy file and the exclusion files used for the simulation (which limits the coverage collection to exactly what is needed at compile-time) can be supplied to VCS UNR.'